torch_xla/distributed/parallel_loader.py

```diff
@@ -95,9 +99,10 @@ def __init__(self,
     thread.daemon = True
     thread.start()
     for dqueue in itervalues(self._queues):
-      thread = threading.Thread(target=self._worker, args=(dqueue,))
-      thread.daemon = True
-      thread.start()
+      for i in range(host_to_device_transfer_threads):
+        thread = threading.Thread(target=self._worker, args=(dqueue,))
+        thread.daemon = True
+        thread.start()
```

Comment:
Can we also modify the resnet example to use this `host_to_device_transfer_threads`? or is that going to cause the accuracy issue compared to the origional implementation?

Reply:
It will only improve the performance and doesn't affect the convergence.